reconciletags/reconciletags.py

```diff
@@ -52,12 +52,16 @@ def reconcile_tags(self, data, dry_run):
         # reconciler.
         self.call('gcloud config list', False)
         for project in data['projects']:
+            default_registry = project['registries'][0]
```

Comment:
Would it make sense to break this out to it's own field?
base_regsitry: "gcr.io"

then additional_registries: [...]